document/js-api/index.bs

```diff
@@ -1039,6 +1054,7 @@ Note: Exported Functions do not have a \[[Construct]] method and thus it is not
 <div algorithm>
 The algorithm <dfn>ToJSValue</dfn>(|w|) coerces a [=WebAssembly value=] to a JavaScript value by performing the following steps:
 
+1. Assert: |w| is not of  the form [=v128.const=] <var ignore>v128</var>.
```

Comment:
Perhaps I'm not understanding this well, why is this assert required? Also, what happens for i64 values without BigInt support?

Reply:
> why is this assert required?

This assert is to ensure that we never call `ToJSValue` with a v128.const, since we don't know how to convert it to JS. In all the callers of `ToJSValue` we should have checked for v128 and raise an exception prior.

> Also, what happens for i64 values without BigInt support?

I think BigInt support is required, not too sure.